apps/api-extractor/src/aedoc/ModuleDocComment.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as ts from 'typescript';
+
+import type { Collector } from '../collector/Collector';
+import { ExtractorMessageId } from '../api/ExtractorMessageId';
+
+export class ModuleDocComment {
```

Comment:
DRY - this is essentially a copy+paste of the code+comments from PackageDocComment.ts, introducing a second full scan of the AST. And it does not conider a comment block containing both `@module` and `@packageDocumentation`, which probably should be an error. I feel like probably these tags should be detected in a single unified scan. But we need to generalize it a bit, because the `@packageDocumentation` code assumed that it can only appear in the entry point file, whereas `@module` can appear in any file.